### DISCUSSION TITLE
Explanation for why the sp_dc cookie is needed

### DISCUSSION THREAD

**Discussion by QuestioningEspecialy** [KRTirtho/spotube#1218] [Q&A]
It's a bit sus considering I don't even know what that cookie's for or what the risks are, so y'all should really just explain it on the Readme and in the software.

**Comment by Ocean-OS** [comment]
The cookies are required so Spotube can retrieve your account's playlists, artists, etc, and so it can allow you to do actions to your playlists or songs as if you were on Spotify, such as adding songs, liking songs, making playlists, etc. The cookies basically let Spotube use your account with Spotify's API.
